src/alt.js

```diff
@@ -77,7 +99,8 @@ class AltStore {
         )
       }
       if (model[LISTENERS][payload.action]) {
-        this[SET_STATE] = false
+        this[IS_DISPATCHING] = true
```

Comment:
do we really need this? can't we use `dispatcher.isDispatching()` to check if it is currently dispatching instead?